### PR TITLE
Read file function compelete,  test not complete 

### LINKED ISSUE #2: Read a file and format into a String
Create a function that reads a .txt file given a filepath as a string. The function should return a string with the contents of the .txt file.

### PR DESCRIPTION
**Pull request by edouarde1**
VSCode Unit testing is not working on my local machine at the moment. Here is a completed text file function. Some re-organization and readme updates.

### CODE REVIEW

**Review by LoganParker — CHANGES_REQUESTED** [review]
We should try to avoid including editor settings like the .vscode folder, and instead add those to the git ignore

File: preprocess.py
Comment (vnajack) [inline preprocess.py:7]: for our project, we should decide on how we should do spacing so that we are all consistent :)

**Review by vnajack — APPROVED** [review]
Looks good to start us off! Thanks!

File: preprocess.py
Comment (edouarde1) [reply preprocess.py:7]: Yeah that would a good idea! Maybe we format that once we get more stuff pushed and merged into master

**Review by LoganParker — APPROVED** [review]
Updated version looks great!

**Pull request merged**
